src/index.ts

```diff
@@ -16,7 +16,7 @@ import { exec, execShell } from "./exec";
 import { checkWorkingDirectory, semverCompare } from "./utils";
 import { getPackageManager } from "./packageManagers";
 
-const DEFAULT_WRANGLER_VERSION = "3.13.2";
+const DEFAULT_WRANGLER_VERSION = "3.x";
```

Comment:
`latest` would potentially introduce a breaking chance once 4.x is out though, wouldn't it, if people are just relying on the defaults?

Perhaps looking up the versions on npm and pulling the latest 3.x from there would be best as a first step? If there was a `v3` tag, it could be as simple as (pseudo-example) `curl https://registry.npmjs.org/wrangler | jq '.["dist-tags"].v3'`.

Reply:
Yeah I briefly played around with looking up the version but it is a bit messy especially if there is no dist-tag, since we would have to iterate over all the versions and find one that best matches the version range.